relay-event-schema/src/protocol/span.rs

```diff
@@ -677,6 +677,9 @@ pub struct SpanData {
     #[metastructure(field = "lcp.url")]
     pub lcp_url: Annotated<String>,
 
+    #[metastructure(field = "span.kind")]
+    pub span_kind: Annotated<String>,
```

Comment:
Please add a small doc string, ideally what values it can have and when it should be filled.

Reply:
See [opentelemetry-proto](https://github.com/open-telemetry/opentelemetry-proto/blob/d7770822d70c7bd47a6891fc9faacc66fc4af3d3/opentelemetry/proto/trace/v1/trace.proto#L180-L183):

```
  // Distinguishes between spans generated in a particular context. For example,
  // two spans with the same name may be distinguished using `CLIENT` (caller)
  // and `SERVER` (callee) to identify queueing latency associated with the span.
```